### PR TITLE
chore(prettier): More formatting cleanup in mdx

### PR DESCRIPTION
**Pull request by evanpurkhiser**

### CODE REVIEW

**Review by kamilogorek — APPROVED** [review]

File: src/docs/product/accounts/quotas/spend-allocation.mdx [hunk 1]
```diff
@@ -25,7 +25,9 @@ To get started with Spend Allocation, look at your total reserved volume and dec
 For each project, you'll need to specify how much of your allocation goes to errors, to transactions, and to attachments.
 
 <Note>
-With Spend Allocation, excess event consumption may come from your <a class href="https://docs.sentry.io/product/accounts/pricing/#on-demand-capacity">on-demand budget</a>, <b>even if you still have reserved volume available.</b>
+
+With Spend Allocation, excess event consumption may come from your [on-demand budget](product/accounts/pricing/#on-demand-capacity), <b>even if you still have reserved volume available.</b>
```
Comment (kamilogorek) [inline src/docs/product/accounts/quotas/spend-allocation.mdx:29]:
```suggestion
With Spend Allocation, excess event consumption may come from your [on-demand budget](/product/accounts/pricing/#on-demand-capacity), <b>even if you still have reserved volume available.</b>
```

File: src/platform-includes/getting-started-install/javascript.angular.mdx [hunk 2]
```diff
@@ -25,8 +25,7 @@ Because of the way Angular libraries are compiled, you need to use a specific ve
 | Older versions  | See note below         |
 
 <Alert level="warning">
-  Support for any Angular version below 10 was discontinued in version 7 of the SDK.
-  If you need to use Angular 9 or older, try version 6 (<code>@sentry/angular@^6.x</code>) of the SDK.
-  For AngularJS/1.x, use <code>@sentry/browser@^6.x</code> and our <a href="./angular1">AngularJS integration</a>.
-  Note, that these versions of the SDK are no longer maintained or tested.
+
+Support for any Angular version below 10 was discontinued in version 7 of the SDK. If you need to use Angular 9 or older, try version 6 (`@sentry/angular@^6.x`) of the SDK. For AngularJS/1.x, use `@sentry/browser@^6.x` and our [AngularJS integration](/angular1). Note, that these versions of the SDK are no longer maintained or tested.
```
Comment (kamilogorek) [inline src/platform-includes/getting-started-install/javascript.angular.mdx:29]:
```suggestion
Support for any Angular version below 10 was discontinued in version 7 of the SDK. If you need to use Angular 9 or older, try version 6 (`@sentry/angular@^6.x`) of the SDK. For AngularJS/1.x, use `@sentry/browser@^6.x` and our [AngularJS integration](/platforms/javascript/guides/angular/angular1). Note, that these versions of the SDK are no longer maintained or tested.
```

**Review by kamilogorek — CHANGES_REQUESTED** [review]

**Comment by kamilogorek** [comment]
2 broken links, the rest looks good

File: src/docs/product/accounts/quotas/spend-allocation.mdx (hunk 1 above)
Comment (evanpurkhiser) [reply src/docs/product/accounts/quotas/spend-allocation.mdx:29]: Thanks for catching these!

**Review by gggritso — APPROVED** [review]

**Pull request merged**